CODEOWNERS

```diff
@@ -33,6 +33,10 @@
 /tests/test_pretrained_train.py @andrewpeng02
 /tests/expected/unzipped/double_zipped @vidushiMaheshwari
 
+# Terraform Files
+/dlp-terraform/lambda/source-files/lambda_function.py @thomaschin35 @avayedawadi
```

Comment:
@farisdurrani this file contains errors since some of the devs listed in this file aren't collaborators with write access to the repo.

Reply:
We'll fix this later